packages/bootstrap/extra/dcos_internal_utils/exhibitor.py

```diff
@@ -90,20 +97,24 @@ def wait(master_count_filename):
 
     data = response.json()
 
-    serving = 0
-    leaders = 0
+    serving = []
+    leaders = []
     for node in data:
         if node['isLeader']:
-            leaders += 1
+            leaders.append(node['hostname'])
         if node['description'] == 'serving':
-            serving += 1
+            serving.append(node['hostname'])
+
+    log.info(
+        "Serving hosts: `%s`, leader: `%s`", ','.join(serving), ','.join(leaders))
 
-    if serving != cluster_size or leaders != 1:
+    if len(serving) != cluster_size or len(leaders) != 1:
         msg = 'Expected {} servers and 1 leader, got {} servers and {} leaders'.format(cluster_size, serving, leaders)
```

Comment:
Won't this now print the contents of the serving/leaders arrays instead of their sizes?
cc: @colin-msphere @orsenthil

Reply:
My bad, sorry about that.

https://github.com/mesosphere/dcos-enterprise/pull/1967
https://github.com/dcos/dcos/pull/2264

Reply:
PRs for `1.9` and `master` have been updated as well.